packages/shared/components/Drawer.svelte

```diff
@@ -104,7 +104,7 @@
     }
 
     async function handleSlideMove(event: CustomEvent): Promise<void> {
-        if ($coords.y < 0) return;
+        if ($coords.y < 0) return
```

Comment:
Let's give `return` the respect it deserves :D. We started writing it inbetween brackets as follows: 
```
if (...) {
  return
}
```

Reply:
Indeed, just added some love to returns.